### DISCUSSION TITLE
how to appoint the database for kudu plugin

### DISCUSSION THREAD

**Discussion by allenchen0314** [apache/seatunnel#1208] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### What happened

![image](https://user-images.githubusercontent.com/81660436/152945400-4f3e171a-b465-48ef-a837-18d37e7557e1.png)
can't appoint the database

### SeaTunnel Version

1.5.7

### SeaTunnel Config

```conf
1
```

### Running Command

```shell
1
```

### Error Exception

```log
1
```

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by yx91490** [comment]
kudu doesn't have the database concept. does your the kudu table created by impala? generally in the format "impala::db.table"

**Reply by allenchen0314** [reply]
> kudu doesn't have the database concept. does your the kudu table created by impala? generally in the format "impala::db.table"

![image](https://user-images.githubusercontent.com/81660436/152980332-9a5dde32-027a-4688-b8fc-763d5d1a4ef6.png)
![image](https://user-images.githubusercontent.com/81660436/152980371-b4dfd2b4-b375-4b03-b56b-f7666a5b5d6a.png)
Still can't
